contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcViews.java

```diff
@@ -0,0 +1,143 @@
+          RpcViewConstants.RPC_CLIENT_SERVER_ELAPSED_TIME_MINUTE_VIEW,
+          RpcViewConstants.RPC_SERVER_ERROR_COUNT_MINUTE_VIEW,
+          RpcViewConstants.RPC_SERVER_SERVER_LATENCY_MINUTE_VIEW,
+          RpcViewConstants.RPC_SERVER_SERVER_ELAPSED_TIME_MINUTE_VIEW,
+          RpcViewConstants.RPC_SERVER_REQUEST_BYTES_MINUTE_VIEW,
+          RpcViewConstants.RPC_SERVER_RESPONSE_BYTES_MINUTE_VIEW,
+          RpcViewConstants.RPC_SERVER_REQUEST_COUNT_MINUTE_VIEW,
+          RpcViewConstants.RPC_SERVER_RESPONSE_COUNT_MINUTE_VIEW,
+          RpcViewConstants.RPC_SERVER_UNCOMPRESSED_REQUEST_BYTES_MINUTE_VIEW,
+          RpcViewConstants.RPC_SERVER_UNCOMPRESSED_RESPONSE_BYTES_MINUTE_VIEW,
+          RpcViewConstants.RPC_CLIENT_ERROR_COUNT_HOUR_VIEW,
+          RpcViewConstants.RPC_CLIENT_ROUNDTRIP_LATENCY_HOUR_VIEW,
+          RpcViewConstants.RPC_CLIENT_REQUEST_BYTES_HOUR_VIEW,
+          RpcViewConstants.RPC_CLIENT_RESPONSE_BYTES_HOUR_VIEW,
+          RpcViewConstants.RPC_CLIENT_REQUEST_COUNT_HOUR_VIEW,
+          RpcViewConstants.RPC_CLIENT_RESPONSE_COUNT_HOUR_VIEW,
+          RpcViewConstants.RPC_CLIENT_UNCOMPRESSED_REQUEST_BYTES_HOUR_VIEW,
+          RpcViewConstants.RPC_CLIENT_UNCOMPRESSED_RESPONSE_BYTES_HOUR_VIEW,
+          RpcViewConstants.RPC_CLIENT_SERVER_ELAPSED_TIME_HOUR_VIEW,
+          RpcViewConstants.RPC_SERVER_ERROR_COUNT_HOUR_VIEW,
+          RpcViewConstants.RPC_SERVER_SERVER_LATENCY_HOUR_VIEW,
+          RpcViewConstants.RPC_SERVER_SERVER_ELAPSED_TIME_HOUR_VIEW,
+          RpcViewConstants.RPC_SERVER_REQUEST_BYTES_HOUR_VIEW,
+          RpcViewConstants.RPC_SERVER_RESPONSE_BYTES_HOUR_VIEW,
+          RpcViewConstants.RPC_SERVER_REQUEST_COUNT_HOUR_VIEW,
+          RpcViewConstants.RPC_SERVER_RESPONSE_COUNT_HOUR_VIEW,
+          RpcViewConstants.RPC_SERVER_UNCOMPRESSED_REQUEST_BYTES_HOUR_VIEW,
+          RpcViewConstants.RPC_SERVER_UNCOMPRESSED_RESPONSE_BYTES_HOUR_VIEW);
+
+  /**
+   * Registers all standard cumulative views.
+   *
+   * <p>This must be called once at the beginning of the process.
+   *
+   * @since 0.11.0
+   */
+  public static void registerCumulativeViews() {
+    registerCumulativeViews(Stats.getViewManager());
+  }
+
+  @VisibleForTesting
+  static void registerCumulativeViews(ViewManager viewManager) {
+    for (View view : RPC_CUMULATIVE_VIEWS_SET) {
+      viewManager.registerView(view);
+    }
+  }
+
+  /**
+   * Registers all standard interval views.
+   *
+   * <p>This must be called once at the beginning of the process.
+   *
+   * @since 0.11.0
+   */
+  public static void registerIntervalViews() {
+    registerIntervalViews(Stats.getViewManager());
+  }
+
+  @VisibleForTesting
+  static void registerIntervalViews(ViewManager viewManager) {
+    for (View view : RPC_INTERVAL_VIEWS_SET) {
+      viewManager.registerView(view);
+    }
+  }
+
+  /**
+   * Registers all views.
+   *
+   * <p>This must be called once at the beginning of the process.
```

Comment:
nit: might also add a note here this method is equivalent to registerCumulativeViews + registerIntervalViews so users don't need to call them again.

Reply:
done.